relayer-cli/src/commands/create/channel.rs

```diff
@@ -17,43 +20,63 @@ use crate::conclude::{exit_with_unrecoverable_error, Output};
 use crate::prelude::*;
 use ibc_relayer::config::default::connection_delay;
 
+static PROMPT: &str = "Are you sure you want a new connection & clients to be created? Hermes will use default security parameters.";
+static HINT: &str = "Consider using the default invocation\n\nhermes create channel --port-a <PORT-ID> --port-b <PORT-ID> <CHAIN-A-ID> <CONNECTION-A-ID>\n\nto re-use a pre-existing connection.";
+
+/// The data structure that represents all the possible options when invoking
+/// the `create channel` CLI command.
+///
+/// There are two possible ways to invoke this command:
+///
+/// `create channel --port-a <Port-ID> --port-b <Port-ID> <Chain-A-ID> <Connection-ID>` is the default
+/// way in which this command should be used, specifying a `Connection-ID` for this new channel
+/// to re-use. The command expects that `Connection-ID` is associated with chain A.
+///
+/// `create channel --port-a <Port-ID> --port-b <Port-ID> <Chain-A-ID> <Chain-B-ID> --new-client-connection`
+/// to indicate that a new connection/client pair is being created as part of this new channel.
+/// This brings up an interactive yes/no prompt to ensure that the operator at least
+/// considers the fact that they're initializing a new connection with the channel.
+///
+/// Note that `Connection-ID`s have to be considered based off of the chain's perspective. Although
+/// chain A and chain B might refer to the connection with different names, they are actually referring
+/// to the same connection.
 #[derive(Clone, Command, Debug, Parser)]
 #[clap(disable_version_flag = true)]
 pub struct CreateChannelCommand {
     #[clap(
         required = true,
-        help = "identifier of the side `a` chain for the new channel"
+        help = "Identifier of the side `a` chain for the new channel"
     )]
     chain_a_id: ChainId,
 
-    #[clap(help = "identifier of the side `b` chain for the new channel (optional)")]
-    chain_b_id: Option<ChainId>,
-
     #[clap(
         short,
         long,
-        help = "identifier of the connection on chain `a` to use in creating the new channel"
+        help = "Identifier of the side `b` chain for the new channel"
     )]
+    chain_b_id: Option<ChainId>,
```

Comment:
Is it consistent that we have `--chain-b-id`, but `--port-b`? I would drop all `_id` suffixes from option names everywhere.